### PR TITLE
Fix dependencies of web wallet example

### PR DESCRIPTION
**Pull request by limemloh**
## Purpose

The web-wallet example is missing some dependencies and importing files outside of its scope.
This PR fixes that.
Closes https://github.com/Concordium/concordium-node-sdk-js/issues/354

### CODE REVIEW

File: examples/wallet/src/util.ts [hunk 1]
```diff
@@ -34,7 +34,6 @@ import {
     nodePort,
     walletProxyBaseUrl,
 } from './constants';
-import { filterRecord, mapRecord } from '../../../packages/sdk/lib/esm/util';
```
Comment (soerenbf) [inline examples/wallet/src/util.ts:37]: Wow... that also seems like a bit of a hack 😲

File: examples/wallet/package.json [hunk 2]
```diff
@@ -17,6 +18,7 @@
         "@vitejs/plugin-react-swc": "^3.4.0",
         "eslint": "^8.50.0",
         "typescript": "^5.2.2",
+        "ts-node": "10.9",
```
Comment (soerenbf) [inline examples/wallet/package.json:21]: Why is this needed? For vite to understand config files in typescript or?

**Review by soerenbf — APPROVED** [review]

File: examples/wallet/package.json (hunk 2 above)
Comment (limemloh) [reply examples/wallet/package.json:21]: Exactly, didn't want to start a fight with the entire configuration, so I just added it here

File: examples/wallet/src/util.ts (hunk 1 above)
Comment (limemloh) [reply examples/wallet/src/util.ts:37]: yes, probably some auto-completion was the cause of this
